packages/compass-e2e-tests/tests/oidc.test.ts

```diff
@@ -169,58 +182,108 @@ describe('OIDC integration', function () {
     if (tmpdir) await fs.rmdir(tmpdir, { recursive: true });
   });
 
-  it('can successfully connect with a connection string', async function () {
-    let tokenFetchCalls = 0;
-    getTokenPayload = () => {
-      tokenFetchCalls++;
-      return DEFAULT_TOKEN_PAYLOAD;
-    };
-    await browser.connectWithConnectionString(connectionString);
-    const result: any = await browser.shellEval(
-      'db.runCommand({ connectionStatus: 1 }).authInfo',
-      true
-    );
-
-    expect(tokenFetchCalls).to.equal(1); // No separate request from the shell
-    expect(result).to.deep.equal(DEFAULT_AUTH_INFO);
-  });
+  describe('with test browser', function () {
+    before(function () {
+      process.env.COMPASS_TEST_OIDC_BROWSER_DUMMY = getTestBrowserCommand();
+      `${
+        process.execPath
+      } ${path.resolve(__dirname, '..', 'fixtures', 'curl.js')}`;
+    });
+
+    after(function () {
+      delete process.env.COMPASS_TEST_OIDC_BROWSER_DUMMY;
```

Comment:
Fwiw, I imagined that this env var/the corresponding bit in the data-service package would go away with this setting, and that we’d just always use the setting instead … does that seem reasonable to you?

Reply:
Missed that TODO, makes sense, updated. Thanks for noticing!